src/comments.js

```diff
@@ -1,17 +1,69 @@
 import dialogPolyfill from 'dialog-polyfill';
+import {
+  getComments, postComments,
+} from './api';
 
 const dialog = document.querySelector('#dialog');
 dialogPolyfill.registerDialog(dialog);
 
-const displayComments = () => {
+const commentsList = document.querySelector('#comments-list');
+
+const displayError = () => {
+  const commentMessage = document.createElement('li');
+  commentMessage.className = 'no-comments';
+  commentMessage.textContent = 'No Comments';
+  commentsList.append(commentMessage);
+};
+
+const displayComment = (comment) => {
+  const commentElement = document.createElement('li');
+  commentElement.className = 'comment';
+  commentElement.textContent = `${comment.creation_date} ${comment.username} : ${comment.comment}`;
+  commentsList.append(commentElement);
+};
+
+const displayComments = async (showId) => {
+  commentsList.innerHTML = '';
+
+  try {
+    const comments = await getComments(showId);
+    comments.forEach((comment) => displayComment(comment));
+  } catch (error) {
+    displayError();
+  }
+};
+
+const inputComment = async (showId) => {
+  const nameInput = document.querySelector('#name');
+  const commentInput = document.querySelector('#comment-input');
+
+  const commentObj = {
+    item_id: `${showId}`,
+    username: nameInput.value.trim(),
+    comment: commentInput.value.trim(),
+  };
+  nameInput.value = '';
+  commentInput.value = '';
+  dialog.close();
+  await postComments(commentObj);
```

Comment:
I would suggest that you clear your inputs after you have run the "postComments(commentObj)" request in order to prevent posting empty values.